CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
 ## Change Log
-All notable changes to this project will be documented in this file.  
+All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased][unreleased]
 - Fixed an issue where symlinks were follwed when removing a module, causing files in vendor to be removed
+- Changes the way how gitignore files are being processed. Retains the layout and other duplicates (comments, empty lines, etc.)
```

Comment:
Small typo there. Should be `Changes the way gitignore files are being processed`.